sonar-project.properties

```diff
@@ -0,0 +1,51 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-WARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# sonar.properties
+#
+# The configuration information to be taken in order to generate the information
+# in Sonar. The project key will be com.telefonica.iot:skuld
+#
+# __author__ = fla
+
+sonar.projectName=Skuld
+sonar.projectKey=com.telefonica.iot:skuld
+sonar.sources=./skuld
```

Comment:
It this source path correct? Or should it be like this ?
  
     sonar.sources=./